lambdas/vonage_api_ingestion/main.py

```diff
@@ -147,26 +134,24 @@ def create_list_of_call_dates(start_date, end_date):
     start_call_date = date.fromisoformat(start_date)
 
     start_call_date = start_call_date + relativedelta(days=+1)
-    # print(f'Date Start {str(start_call_date)}')
 
     end_call_date = date.fromisoformat(end_date)
-    # print(f'Date End {str(end_call_date)}')
 
     date_counter = start_call_date
     between_dates = []
     while date_counter < end_call_date:
-        # print(f'Date Added {str(date_counter)}')
         between_dates.append(str(date_counter))
         date_counter = date_counter + relativedelta(days=+1)
     return between_dates
-    # read the dates as actual dates
 
 
 def single_digit_to_zero_prefixed_string(value):
     return str(value) if value > 9 else '0' + str(value)
```

Comment:
you don't really need to do this you can use the built in zfill() or formatting with f strings
[https://docs.python.org/3/library/stdtypes.html#str.zfill](https://docs.python.org/3/library/stdtypes.html#str.zfill)

value.zfill(2)